src/widgets/sparkline.rs

```diff
@@ -253,6 +253,7 @@ mod tests {
 
     #[test]
     fn it_does_not_panic_if_max_is_set_to_zero() {
+        #[expect(clippy::unnecessary_min_or_max)]
```

Comment:
expect is 1.8.1, which would raise out MSRV too high for now. I'd like to keep this at least 2 versions back to give people a little time to upgrade (this aligns with the policy of the `time` crate)
```suggestion
        #[allow(clippy::unnecessary_min_or_max)]
```
Also, I think this might be a slightly different form of https://github.com/rust-lang/rust-clippy/issues/13191